docs/source/gpu.rst

```diff
@@ -0,0 +1,162 @@
+GPUs
+====
+
+Dask works with GPUs in a few ways.
+
+
+Custom Computations
+-------------------
+
+Many people use Dask alongside GPU-accelerated libraries like PyTorch and
+TensorFlow to manage workloads across several machines.  They typically use
+Dask's custom APIs, notably :doc:`Delayed <delayed>` and :doc:`Futures
+<futures>`.
+
+Dask doesn't need to know that these functions use GPUs.  It just runs Python
+functions.  Whether or not those Python functions use a GPU is orthogonal to
+Dask.  It will work regardless.
+
+As a worked example, you may want to view this talk:
+
+.. raw:: html
+
+    <iframe width="560"
+            height="315"
+            src="https://developer.download.nvidia.com/video/gputechconf/gtc/2019/video/S9198/s9198-dask-and-v100s-for-fast-distributed-batch-scoring-of-computer-vision-workloads.mp4"
+            frameborder="0"
+            allow="autoplay; encrypted-media"
+            allowfullscreen>
+    </iframe>
+
+
+
+High Level Collections
+----------------------
+
+Dask can also help to scale out large array and dataframe computations by
+combining the Dask Array and DataFrame collections with a GPU-accelerated
+array or dataframe library.
+
+Recall that :doc:`Dask Array <array>` creates a large array out of many NumPy
+arrays and :doc:`Dask DataFrame <dataframe>` creates a large dataframe out of
+many Pandas dataframes.  We can use these same systems with GPUs if we swap out
+the NumPy/Pandas components with GPU-accelerated versions of those same
+libraries, as long as the GPU accelerated version looks enough like
+NumPy/Pandas in order to interoperate with Dask.
+
+Fortunately, libraries that mimic NumPy, Pandas, and Scikit-Learn on the GPU do
+exist.
+
+
+DataFrames
+~~~~~~~~~~
+
+The `RAPIDS <https://rapids.ai>`_ libraries provide a GPU accelerated
+Pandas-like library,
+`cuDF <https://rapidsai.github.io/projects/cudf/en/latest/>`_,
+which interoperates well and is tested against Dask DataFrame.
+
+If you have cudf installed then you should be able to convert a Pandas-backed
+Dask DataFrame to a cuDF-backed Dask DataFrame as follows:
+
+.. code-block:: python
+
+   import cudf
+
+   df = df.map_partitions(cudf.from_pandas)  # convert pandas partitions into cudf partitions
+
+However, cuDF does not support the entire Pandas interface, and so a variety of
+Dask DataFrame operations will not function properly.
+
+
+Arrays
+~~~~~~
+
+.. note:: Dask's integration with CuPy relies on features recently added to
+   NumPy and CuPy, particularly in version ``numpy>=1.17`` and ``cupy>=6``
```

Comment:
Using ``numpy>=1.16`` works too, although it's necessary to set the `NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=1` environment variable. I see that perhaps you're trying to avoid referencing the environment variable, but there are two issues to note:

1. NumPy 1.17 has not been released yet;
1. Despite the environment variable having been removed, I remember seeing some discussions that this may still be added back even for 1.17.

Reply:
>  I see that perhaps you're trying to avoid referencing the environment variable

Indeed, especially for a high level document like this.

> NumPy 1.17 has not been released yet;

I'm ok with that.

> Despite the environment variable having been removed, I remember seeing some discussions that this may still be added back even for 1.17.
 
That's unfortunate.